net-dialup/tio/tio-2.5.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="A simple serial device I/O tool"
+DESCRIPTION="tio is a simple serial device tool which features a straightforward command-line and configuration file interface to easily connect to serial TTY devices for basic I/O operations."
+HOMEPAGE="https://tio.github.io/"
+COPYRIGHT="2014-2022  Martin Lund"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/tio/tio/releases/download/v$portVersion/tio-$portVersion.tar.xz"
+CHECKSUM_SHA256="063952ee90a78cee180f6e660d6c73773dfc109efcdc151585accfe1500c44a7"
+SOURCE_FILENAME="tio-$portVersion.tar.xz"
+SOURCE_DIR="tio-$portVersion"
+PATCHES="tio-2.5.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
```

Comment:
Not needed, by default meson installs binaries in $prefix/bin (arch independent):
```diff
-commandBinDir=$binDir
-commandSuffix=$secondaryArchSuffix
-if [ "$targetArchitecture" = x86_gcc2 ]; then
-	commandBinDir=$prefix/bin
-	commandSuffix=
-fi
```

Reply:
As the wiki mentions, it's only needed when you want to change the path for --bindir, which in this case isn't needed, so there isn't realy anything that needs updating there (imho). :)
For `meson` this is always `$prefix/bin`, for autotools with `runConfigure` this turns into `$binDir`.
A thing to be added could be for the cmake builds when using `$cmakeDirArgs` (does the same as `runConfigure` for autotools) where `-DCMAKE_INSTALL_BINDER` leads to `$binDir` if not otherwise set.

Reply:
Updated some information about this on my wiki: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#commandbindir-and-commandsuffix